src/components/contact/contactform.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import emailjs from 'emailjs-com';
+import './contactForm.css';
+
+const ContactForm = () => {
+    const [formStatus, setFormStatus] = useState('');
+
+    const sendEmail = (e : React.FormEvent) => {
+        e.preventDefault();
+
+        emailjs.sendForm('service_wqswvbr', 'template_ot5okin', e.target as HTMLFormElement, '7fupPFCmNa5f-tNUl')
```

Comment:
Is that some kind of app secret? If so you should look up a better way to handle that instead of just having it out in the JS like that publicly visible. Not sure if it applies here since it's the public key but still keep that in mind always with secrets in JS